modules/creating-manifest-file-customized-br-ex-bridge.adoc

```diff
@@ -0,0 +1,218 @@
+
+This feature supports the following tasks:
+
+* Modifying the maximum transmission unit (MTU) for your cluster.
+* Modifying attributes of a different bond interface, such as MIImon (Media Independent Interface Monitor), bonding mode, or Quality of Service (QoS).
+* Updating DNS values.
+endif::postinstall-bare-metal-ipi,postinstall-bare-metal-upi[]
+
+Consider the following use cases for creating a manifest object that includes a customized `br-ex` bridge:
+
+* You want to make postinstallation changes to the bridge, such as changing the Open vSwitch (OVS) or OVN-Kubernetes `br-ex` bridge network. The `configure-ovs.sh` shell script does not support making postinstallation changes to the bridge.
+* You want to deploy the bridge on a different interface than the interface available on a host or server IP address.
+* You want to make advanced configurations to the bridge that are not possible with the `configure-ovs.sh` shell script. Using the script for these configurations might result in the bridge failing to connect multiple network interfaces and facilitating data forwarding between the interfaces.
+
+ifndef::postinstall-bare-metal-ipi,postinstall-bare-metal-upi[]
+
+[NOTE]
+====
+If you require an environment with a single network interface controller (NIC) and default network settings, use the `configure-ovs.sh` shell script.
+====
+
+After you install {op-system-first} and the system reboots, the Machine Config Operator injects Ignition configuration files into each node in your cluster, so that each node received the `br-ex` bridge network configuration. To prevent configuration conflicts, the `configure-ovs.sh` shell script receives a signal to not configure the `br-ex` bridge.
+endif::postinstall-bare-metal-ipi,postinstall-bare-metal-upi[]
+
+.Prerequisites
+ifndef::postinstall-bare-metal-ipi,postinstall-bare-metal-upi[]
+* Optional: You have installed the link:https://nmstate.io/[`nmstate`] API so that you can validate the NMState configuration.
+endif::postinstall-bare-metal-ipi,postinstall-bare-metal-upi[]
+
+ifdef::postinstall-bare-metal-ipi,postinstall-bare-metal-upi[]
+* You used the `configure-ovs.sh` shell script to create a `br-ex`.
+* You install the Kubernetes NMState Operator.
+endif::postinstall-bare-metal-ipi,postinstall-bare-metal-upi[]
+
+.Procedure
+
+ifndef::postinstall-bare-metal-ipi,postinstall-bare-metal-upi[]
+. Create a NMState configuration file that has decoded base64 information for your customized `br-ex` bridge network:
++
+.Example of an NMState configuration for a customized `br-ex` bridge network
+[source,yaml]
+----
+interfaces:
+- name: enp2s0 <1>
+  type: ethernet <2>
+  state: up <3>
+  ipv4:
+    enabled: false <4>
+  ipv6:
+    enabled: false
+- name: br-ex
+  type: ovs-bridge
+  state: up
+  ipv4:
+    enabled: false
+    dhcp: false
+  ipv6:
+    enabled: false
+    dhcp: false
+  bridge:
+    port:
+    - name: enp2s0 <5>
+    - name: br-ex
+  - name: br-ex
```

Comment:
There is indentation error in Line98.

Reply:
Thanks. I removed the indent from line 98 so it aligns with line 85.